test/quantization/core/test_quantized_module.py

```diff
@@ -396,14 +396,16 @@ def _test_conv_api_impl(
             qconv_module, [example_input_q],
             check_save_load=True)
 
-        if post_op == "add":
-            # **TODO Leslie** Remove this part when enabling the lowering in next PR.
-            # workaround in this PR to return from here, since the below lowering part enabled in next PR
-            # We will enable below check in next PR
-            return
+        class _FusedModule_two_input_args(torch.nn.intrinsic._FusedModule):
```

Comment:
do we need this?

Reply:
Yes, I think so. That's because original `torch.nn.intrinsic._FusedModule` only support one input.